electron/src/js/menu/system.ts

```diff
@@ -399,6 +399,20 @@ const createMenu = (isFullScreen: boolean): Menu => {
     helpTemplate.submenu = [];
   }
 
+  // Global mute shortcut
+  globalShortcut.register('CmdOrCtrl+Alt+M', () => sendAction(EVENT_TYPE.CONVERSATION.TOGGLE_MUTE));
+
+  // Global account switching shortcut
+  const switchAccountShortcut = ['CmdOrCtrl', 'Super'];
+  const accountLimit = 3;
```

Comment:
Is the account limit specified somewhere else? I could imagine we have it as a config setting already?